adapter/internal/oasparser/envoyconf/envoyconf_internal_test.go

```diff
@@ -485,44 +476,44 @@ func TestGenerateRegexSegment(t *testing.T) {
 	dataItems := []generateRegexSegmentTestItem{
 		{
 			inputPath:    "/v2/pet/",
-			regexSegment: "/v2/pet(/{0,1})",
+			regexSegment: "/v2/pet/",
```

Comment:
I guess the trailing slash is there because you used the newly introduced method which only does the path parameter replacement.

Reply:
yes. This covers the method `replacePathParamsWithCaptureGroups` which only does the path parameter replacement.